platform/wayland/cog-platform-wl.c

```diff
@@ -524,42 +557,6 @@ registry_on_global(void *data, struct wl_registry *registry, uint32_t name, cons
     g_debug("%s '%s' interface obtained from the Wayland registry.", interface_used ? "Using" : "Ignoring", interface);
 }
 
-#ifdef COG_USE_WAYLAND_CURSOR
-void
-cog_wl_platform_set_cursor(CogWlPlatform *platform, enum cursor_type type)
```

Comment:
This is renamed as cog_wl_seat_set_cursor:  https://github.com/Igalia/cog/pull/623/commits/a321b21181571ea271b3465d181e2581e9bc38d7#diff-b5c8d7e31c380d86e8ae8aea3f2dc76fd5f407590ccc4a6cf2347ca0b9e17749R249